lib/matplotlib/scale.py

```diff
@@ -545,6 +623,7 @@ def limit_range_for_scale(self, vmin, vmax, minpos):
     'log':    LogScale,
     'symlog': SymmetricalLogScale,
     'logit':  LogitScale,
+    'arbitrary': ArbitraryScale,
```

Comment:
It's a bit misleading that `LogTransformBase` can have an `ArbitraryScale`.

Reply:
Not following this comment...

Reply:
Sorry. Never mind. I mesread the code.